lib/topological_inventory/ansible_tower/messaging_client.rb

```diff
@@ -11,14 +11,13 @@ def worker_listener
         @worker_listener ||= ManageIQ::Messaging::Client.open(worker_listener_opts)
       end
 
-      def targeted_refresh_listener
-        @targeted_refresh_listener ||= ManageIQ::Messaging::Client.open(targeted_refresh_listener_opts).tap do |client|
-          # persistent workers by pod hostname, this will prevent rebalances.
-          client.send(:kafka_client)[:"group.instance.id"] = ENV['HOSTNAME']
-
-          # 30 second timeout, after this the worker of the old hostname (ie in the event of a redeploy) will be removed and the topic will be rebalanced.
-          client.send(:kafka_client)[:"session.timeout.ms"] = 30 * 1000
```

Comment:
the problem is related to `max.poll.interval.ms`, so I believe this can't help too

Reply:
Yeah I'm not sure whats going on why they would just freeze like this. This would increase the time a client has to process a message etc, I'm ok with reverting it for now. But in the future if we're seeing constant disconnects/reconnects this setting might help us.